MyLibrary/Sources/DataClient/Client.swift

```diff
@@ -50,6 +52,108 @@ extension DataClient: DependencyKey {
   }
 }
 
+extension DependencyValues {
+  public var dataClient: DataClient {
+    get { self[DataClient.self] }
+    set { self[DataClient.self] = newValue }
+  }
+}
```

Comment:
I think we don't have to back to KeyPath style.

Reply:
I found how to set withDependencies in the DependencyClient style. 😮 
https://github.com/pointfreeco/swift-dependencies/discussions/175

I fixed it.